### ISSUE TITLE
Error with color map node not connecting to image texture node.

### ISSUE THREAD

**Issue by hassan-hsh** [blendernc/blendernc-zip-install#3]
Hi,
I've been following along with the documentation, installed the python modules to my Blender 4.0 folder, and then started following the Gebco topography example and have this error:
`Python: Traceback (most recent call last):
  File "C:\Users\name\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\blendernc\operators.py", line 302, in execute
    blendernc_material.node_tree.links.new(cmap.inputs[0], imagetex.outputs[0])
IndexError: bpy_prop_collection[index]: index 0 out of range, size 0
`
I then checked the nodes in the shading tab and it seems the color map node is not connected to the image texture node?
I also attempted the same thing in Blender 3.4 but had no success (it didn't recognise any variables in the NC file)
Any ideas what could be wrong?

**Comment by josuemtzmo** [comment]
Hello @hassan-hsh, 
It should work in Blender 3.XX. I haven't updated the code to support Blender 4.

Are you getting the same error in Blender 3.4? If you have no variables in the NC node, It's likely that there was an issue in the installation of the python modules, particularly `xarray`.

**Comment by hassan-hsh** [comment]
Ah yes, I see that xarray hasn't been installed.
This is the .bat file I'm running to install modules: 
```
set BLENDERPY="C:\Program Files\Blender Foundation\Blender 3.4\3.4\python\bin"
%BLENDERPY%-m ensurepip
%BLENDERPY% -m pip install -r requirements.txt
$BLENDERPY -m pip install numpy cython zarr ecmwflibs cfgrib xarray cmocean matplotlib dask dask scipy netCDF4 toolz pooch psutil
```

**Comment by hassan-hsh** [comment]
hey just thought I'd update that I resolved it. Just was stupidly not putting a path to the actual .exe.

All fixed now running nicely in blender 3.6.

Love this tool I'm excited to see it grow! Thanks so much

**Comment by josuemtzmo** [comment]
Thanks, I'm glad it's working now. If you have any issue, please post directly at: https://github.com/blendernc/blendernc/issues
